### ISSUE TITLE
Fix declaration-block-no-redundant-longhand-properties false positives for position

### ISSUE THREAD

**Issue by rludgero** [stylelint/stylelint#7433]
### What minimal example or steps are needed to reproduce the bug?

**CSS**
```css
.media-wrapper {
    display: block;
    height: inherit;
    margin: 0;
    max-width: 100%;
    overflow: hidden;
    pointer-events: none;
    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    width: inherit;
    z-index: 2;
}
```

**SASS**
```scss
.media {
    &-wrapper {
        display: block;
        height: inherit;
        margin: 0;
        max-width: 100%;
        overflow: hidden;
        pointer-events: none;
        position: absolute;
        top: 0;
        right: 0;
        bottom: 0;
        left: 0;
        width: inherit;
        z-index: 2;
    }
}
```

### What minimal configuration is needed to reproduce the bug?

Windows 10 Pro X64
Node v 20.10.0
Stylelint v.16.1.0
NPM v 10.2.3
SASS v 1.69.5

**stylelintrc.json**
```json
{
    "extends": [
        "stylelint-config-standard-scss"
    ],
    "rules": {
        "declaration-block-no-redundant-longhand-properties": [
            true, {
                "message": "https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#true"
            }
        ],
    }
}
```

### How did you run Stylelint?

**package.json**
```json
"scripts": {
  "css:compiler": "sass src/assets/styles/sass/style.scss src/assets/styles/style.css",
  "lint:css": "cross-env stylelint --cache src/assets/styles/sass/**/index.scss"
}
```

**git-bash for windows**
```bash
npm run lint:css
```

### Which Stylelint-related dependencies are you using?

```json
"devDependencies": {
    "cross-env": "^7.0.3",
    "sass": "^1.69.5",
    "stylelint": "^16.1.0",
    "stylelint-config-standard-scss": "^12.0.0"
}
```

### What did you expect to happen?

Expect the linter not detect **position** properties like **top**, **right**, **bottom**, **left** from the **declaration-block-no-redundant-longhand-properties** rule.

### What actually happened?

The **declaration-block-no-redundant-longhand-properties** rule, flag a false positive for position properties

### Do you have a proposal to fix the bug?

Exclude linter for **position** properties like **top**, **right**, **bottom**, **left** from the **declaration-block-no-redundant-longhand-properties** rule.

**Comment by jeddy3** [comment]
@rludgero Thanks for the report.

You can use the [`inset` shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) instead of `top`, `bottom`, `left` and `right`.

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0BbGUCWBDAtAdwCcsAHYmAgAmAB0A7ChizAZ2IBssBPeCgIzYhgA1gG46jCgAsYGAOaSALjwy1pBDArH1GKLAVkqeABi0TdAD3wYoCyTwCMRowFJTjCADdyAMwF4ektawtG4MxBAqCuQ4MF60Csw8tBC0MKEU4cwaGCk8WLzMEGwArlHpChDExunq8koUJuKMvBAKFSjVTQxsMN71jdoMeNa2yqrkGukAXjgqsOY8AExaAL4gADQg3hg9AHJYaAggMOYH7DBIYMzMG+Ap27JHNNrUx+ZRtFDMrzwA2l0UV7MBScHpsSI4SC0B44YFYT56KCvJoAXToa02UIeADEIARdAojgArQq0W6wYg3RDPBhAkFgyI-QEgDhRYGvdZNOmgmDg+KQ+5yWEKeFQRFM16smDskDokArIA).

If you want, you can [configure the rule](https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#ignoreshorthands-regex-regex-string) to ignore the `inset` shorthand:

```json
{
    "extends": [
        "stylelint-config-standard-scss"
    ],
    "rules": {
        "declaration-block-no-redundant-longhand-properties": [
            true, {
                "ignoreShorthands": ["inset"],
                "message": "https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#true"
            }
        ]
    }
}
```

Does that answer your query?

**Comment by rludgero** [comment]
> @rludgero Thanks for the report.
> 
> You can use the [`inset` shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) instead of `top`, `bottom`, `left` and `right`.
> 
> [Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0BbGUCWBDAtAdwCcsAHYmAgAmAB0A7ChizAZ2IBssBPeCgIzYhgA1gG46jCgAsYGAOaSALjwy1pBDArH1GKLAVkqeABi0TdAD3wYoCyTwCMRowFJTjCADdyAMwF4ektawtG4MxBAqCuQ4MF60Csw8tBC0MKEU4cwaGCk8WLzMEGwArlHpChDExunq8koUJuKMvBAKFSjVTQxsMN71jdoMeNa2yqrkGukAXjgqsOY8AExaAL4gADQg3hg9AHJYaAggMOYH7DBIYMzMG+Ap27JHNNrUx+ZRtFDMrzwA2l0UV7MBScHpsSI4SC0B44YFYT56KCvJoAXToa02UIeADEIARdAojgArQq0W6wYg3RDPBhAkFgyI-QEgDhRYGvdZNOmgmDg+KQ+5yWEKeFQRFM16smDskDokArIA).
> 
> If you want, you can [configure the rule](https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#ignoreshorthands-regex-regex-string) to ignore the `inset` shorthand:
> 
> ```json
> {
>     "extends": [
>         "stylelint-config-standard-scss"
>     ],
>     "rules": {
>         "declaration-block-no-redundant-longhand-properties": [
>             true, {
>                 "ignoreShorthands": ["inset"],
>                 "message": "https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#true"
>             }
>         ]
>     }
> }
> ```
> 
> Does that answer your query?

Thanks for the tip, I didn't know that they had created this css property shorthand for positioning.

For the moment, I will use the **ignoreShorthands** option, but it's good to know to use it in a future project. 

Just in case someone else is interested when they consult this issue, I leave this link that I find useful https://css-tricks.com/almanac/properties/i/inset/
